src/main.cpp

```diff
@@ -9188,7 +9186,7 @@ bool NeuralNodeParticipates()
 
 bool StrLessThanReferenceHash(std::string rh)
 {
-    std::string address_day = rh + "_" + RoundToString(GetDayOfYear(),0);
+    std::string address_day = rh + "_" + RoundToString(GetDayOfYear(GetAdjustedTime()),0);
```

Comment:
I've been thinking we should try to get rid of as many `RoundToString` and `cdbl` as possible. Change this line to:
```
std::string address_day = rh + "_" + std::to_string(GetAdjustedTime());
```